tests/scancode/test_cli.py

```diff
@@ -898,3 +905,6 @@ def test_merge_multiple_scans():
     with open(result_file, read_mode) as f:
         result_files = json.loads(f.read())['files']
     assert expected_files == result_files
+
+
+#test_scanned_resource_no_attribute_emails()
```

Comment:
I am not sure what this is: some leftover? may be you can remove these few lines?